### PR TITLE
Update biomodels download URL

### PR DESCRIPTION
**Pull request by alubbock**
The previous EBI link stopped working, which was breaking the unit
tests. I've updated it to the Caltech mirror.

### CODE REVIEW

**Comment by jmuhlich** [comment]
Seems to be working now... Do we believe the Caltech mirror is going to be more stable than the EBI one, in the long term?

**Comment by jmuhlich** [comment]
Actually I just realized the solution for the tests is to mock out the biomodels download. We don't need to test that the biomodels website is working, we only need to test our code that sends the request and parses the response.

**Comment by alubbock** [comment]
I added both Biomodels mirrors as a switchable option, and used a mock to use a local SBML file from BioNetGen for the unit tests.

File: pysb/importers/sbml.py [hunk 1]
```diff
@@ -15,7 +15,10 @@
 from pysb.logging import get_logger, EXTENDED_DEBUG
 
 BIOMODELS_REGEX = re.compile(r'(BIOMD|MODEL)[0-9]{10}')
-BIOMODELS_URL = 'http://www.ebi.ac.uk/biomodels-main/download?mid={}'
+BIOMODELS_URLS = {
+    'EBI': 'http://www.ebi.ac.uk/biomodels-main/download?mid={}',
```
Comment (jmuhlich) [inline pysb/importers/sbml.py]: Could the keys be all lowercase, `'ebi'` and `'caltech'`? That seems more pythonic to me.

**Pull request merged**